apps/web/server/routers/viewer.tsx

```diff
@@ -428,11 +428,6 @@ const loggedInViewerRouter = createProtectedRouter()
         skip,
       });
 
-      const groupedRecurringBookings = await prisma.booking.groupBy({
-        by: [Prisma.BookingScalarFieldEnum.recurringEventId],
-        _count: true,
-      });
-
```

Comment:
As a bonus, we removed this extra query from the bookings query as we now rely on the already returned `bookings` list.

Reply:
Does this is still needed on `apps/web/pages/api/book/confirm.ts` tho? at line 200?

https://github.com/calcom/cal.com/blob/805419d58ba6918a305f66a874a3e2fdddbdad4b/apps/web/pages/api/book/confirm.ts#L198

Reply:
Yes, is needed, as that covers the API side when confirming events to be able to show correctly the number of actual events booked in the email, as a recurring event may allow 10 events and a user can book less than that.